test/specs/lib/FocusZone-test.tsx

```diff
@@ -1110,4 +1110,71 @@ describe('FocusZone', () => {
     expect(inputA.tabIndex).toBe(-1)
     expect(buttonB.tabIndex).toBe(0)
   })
+
+  it('should force focus to first focusable element when FocusZone container receives focus and shouldFocusFirstElementWhenReceivedFocus is set to "true"', () => {
+    const isInnerZoneKeystroke = (e: React.KeyboardEvent<HTMLElement>): boolean =>
+      keyboardKey.getCode(e) === keyboardKey.Enter
+    const isFocusableProperty = { [IS_FOCUSABLE_ATTRIBUTE]: true }
+
+    const component = ReactTestUtils.renderIntoDocument(
+      <div {...{ onFocusCapture: onFocus }}>
+        <FocusZone
+          direction={FocusZoneDirection.horizontal}
+          shouldEnterInnerZone={isInnerZoneKeystroke}
+        >
+          <button className="a">a</button>
```

Comment:
nit: seems that using 'id' attribute would better reflect tests semantics, as for selectors there is **always** one element expected to be returned

Reply:
fair enough, but I think it easier to read tests when it is written

```document.querySelector('.a')```

vs 

```document.querySelector('[id="a"]')```

(```getElementById``` does not exists on ```HTMLElement```)

I think we can be safe here as it is just a plain example, so I would leave it as it is.

Reply:
document.querySelector('#a') - this is a selector that should be used